examples/set-namespace-ns-unset/.expected/config.yaml

```diff
@@ -0,0 +1 @@
+exitCode: 0
```

Comment:
This is the default value. So you can remove this line and then remove this file. It should not affect the tests.

Reply:
thanks!